assets/src/scripts/monitoring-location/components/hydrograph/date-controls.js

```diff
@@ -161,14 +171,14 @@ const drawCustomRadioButtons = function(container, initialDateRange) {
  * @param {D3 selection} container
  * @param {Redux store} store
  * @param {String} siteno
- * @param {String} initialDateRange - Used to set the initial contents of the text field.
+ * @param {String} initialDateRange - Either in the form of P<some number>P(or Y). Used to set the initial contents of the text field.
  */
 const drawCustomDaysBeforeForm = function(container, store, siteno, initialDateRange) {
     const formContainer = container.append('div')
         .attr('id', 'ts-custom-days-before-today-select-container')
         .attr('class', 'usa-form')
         .attr('aria-label', 'Custom date by days before today specification')
-        .attr('hidden', initialDateRange === 'custom' ? true : null);
+        .attr('hidden', STANDARD_INITIAL_VALUES.includes(initialDateRange) ? true : null);
```

Comment:
isn't this the same as !isCustomPeriod(initialDateRange)? If it is, I don't think we need STANDARD_INITIAL_VALUES.

Reply:
Why yes it is. Good point! Using it does result in a number of double negative Booleans, which I always find confusing, but it does make sense to use isCustomPeriod for consistency.